internal/controller/pkg/revision/reconciler.go

```diff
@@ -229,6 +236,14 @@ func WithServiceAccount(sa string) ReconcilerOption {
 	}
 }
 
+// WithRuntimeConfigEnabled specifies whether the Reconciler should enable
+// runtime configuration for package runtime.
+func WithRuntimeConfigEnabled(b bool) ReconcilerOption {
+	return func(r *Reconciler) {
+		r.runtimeConfigEnabled = b
+	}
+}
```

Comment:
It might be more future-proof to just plumb the feature flags down into the reconciler. That way the reconciler can call `o.Features.Enabled(...)` to figure out whether runtime configs are enabled, and also do the same for any other feature flags that it's affected by in future.

Reply:
Done. 
I passed down the features only not the whole options, so, calling as `r.features.Enabled(...)` which I assume what you meant.